package.json

```diff
@@ -10,5 +10,8 @@
   },
   "devDependencies": {
     "husky": "^7.0.4"
+  },
```

Comment:
Zły package json :)

Reply:
ale Husky to nie ja dodawałem

Reply:
W złym miejscu dodałem. Choziło mi o react router

Reply:
Do frontend package/katalogu

Reply:
Mamy 2 package.json. React router dotyczy tylko frontu

Reply:
więc powinien być w `frontend/package.json` dodany
A nie w katalogu głównym

Reply:
nie chcemy aby BE miał dostep do react-router